ext/zend_test/tests/sensitive_parameter.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Test that zend_mark_function_parameter_as_sensitive works.
+--FILE--
```

Comment:
You need to declare that the zend_test extension is required for this test, so that it is attempted to load it dynamically (this ext is built dynamically on Windows by default):
```suggestion
--EXTENSIONS--
zend_test
--FILE--
```

Reply:
Ah, thanks, fixed. I am confused why the other AppVeyor build worked, though.